ultimatejob/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for ultimatejob project.
```

Comment:
wsgi is the standard module for building your website. You can't have both asgi and wsgi in the same project.